Tests/Marketplace/upload_packs.py

```diff
@@ -555,7 +555,6 @@ def main():
             continue
 
         pack.status = PackStatus.SUCCESS.name
-        pack.cleanup()
```

Comment:
@IkaDemisto why do you remove the pack cleanup?

Reply:
No need for it and because of it wrong uploaded version is printed in the summary.